asdf/versioning.py

```diff
@@ -156,7 +156,17 @@ def __hash__(self):
     AsdfVersion("1.6.0"),
 ]
 
-default_version = AsdfVersion("1.5.0")
+""" quote from file:///config.rst:
+default_version
+---------------
+
+The default ASDF Standard version used for new files.  This can be overridden
+on an individual file basis (using the version argument to ``AsdfFile.__init__``)
+or set here to change the default for all new files created in the current session.
+
+Defaults to the latest supported ASDF Standard version.
+"""
```

Comment:
First don't copy/paste quotes from the documentation to the source code. We want to avoid the issues that occur with duplication whenever possible.

Second, in `docs/asdf/config.rst` the following update needs to occur:
```
Defaults to the latest supported ASDF Standard version.
```
should be changed to
```
Defaults to the latest stable ASDF Standard version.
```
This is because ASDF Standard 1.6 is still in flux and has not been fully adopted yet.